.github/workflows/tests.yml

```
@@ -0,0 +1,61 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - "**" # all branches
+      - "!master" # except master
+      - "!develop" # except develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      # TODO: may not be needed, just trying it out (could be cool)
+      matrix:
+        os: [ubuntu-latest]
+        java: [8, 20]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Cache npm dependencies
+        uses: actions/cache@v3
+        with:
+          path: |
+            - ./node_modules
+            - ./selenium/node_modules
+            - ./playwright/node_modules
+          key: npm-workspace-cache-${{ runner.os }}-${{ hashFiles('./**/package-lock.json') }}
+          restore-keys: |
+            npm-workspace-cache-${{ runner.os }}-
+      - name: Cache Maven dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
```

Does the Java version matter? Do dependencies for Java v8 work with Java v20?

```suggestion
          key: ${{ runner.os }}-maven-${{ matrix.java }}-${{ hashFiles('**/pom.xml') }}
```

---

Ya this is a draft (I accidentally opened it up when turned it into a draft so you'll still get spammed in GH notifications :( ) - We can just use Java 8 no need to waste build minutes here